### ISSUE TITLE
QR Code [fun]

### ISSUE THREAD

**Issue by piperfw** [tempoCollaboration/OQuPy#93]
Having a QR code linking to the Repo or documentation is pretty handy for putting on academic posters or leaflets.

Attached generated with the OQuPy logo/colours using https://www.qrcode-monkey.com/

Might put it in `docs/graphics` if it would be useful to others (or we want to standardise it).

![oqupy-qr-code](https://github.com/tempoCollaboration/OQuPy/assets/29196860/92331367-db9b-4179-997a-299cd810cfd8)

**Comment by gefux** [comment]
That looks very nice. Yes, let's add an .SVG and .PNG to docs/graphics.

**Comment by piperfw** [comment]
@gefux I've added the QR code to a few places on a branch qr-docs:

- oqupy-github-qr.png in docs/graphics/
- "Consider sharing this project" plus qr-code added to the bottom of README.md
- New page (sharing.rst) 'Sharing this project' in the docs with download links to logo and qr-code, plus added qr-code to bottom of index.rst
- [renamed How To Cite -> Citing this project to be consistent]

The above were made as separate commits so we can undo as necessary. Let me know what you think.

Everything built locally although I'm slightly nervous about the download links in `sharing.rst`.

**Comment by gefux** [comment]
Sounds great. I'd really would like to have an SVG graphic as well if possible.
Could you make a PR, so we can have a look?

**Comment by piperfw** [comment]
I've generated a new QR Code with .png and .svg using qr-codemonkey.com.

This seemed to accept a .svg for the central logo just fine; but it's possible the tool converts this to raster before embedding in the QR code (if we are concerned about this we could always generate a .svg qr-code with a blank centre and then add the OQuPy cloud afterwards). For reference these are the parameters I used for the online tool:
[qr-parameters.txt](https://github.com/tempoCollaboration/OQuPy/files/14084436/qr-parameters.txt)

PR made.
